process/smartContract/builtInFunctions/esdtNFTTransfer.go

```diff
@@ -231,6 +231,11 @@ func (e *esdtNFTTransfer) processNFTTransferOnSenderShard(
 		return nil, err
 	}
 
+	tokenNonce := esdtData.TokenMetaData.Nonce
```

Comment:
all these changes has to be put inside vm-common. not here. as this is getting deleted.

Reply:
done.